lib/manifest.js

```diff
@@ -0,0 +1,15 @@
+const deepAssign = require('deep-assign');
+const fs = require('fs-extra');
+
+module.exports.merge = (filename, data) => {
+  return fs.readJson(filename).then(manifest => {
+    return deepAssign({}, manifest, data);
```

Comment:
I forget what happens with arrays in deep, recursive object merges.

can you test with a`package.json` with multiple videos already defined in the package? do the new ones get appended or replaced?